templates/debug.hbs

```diff
@@ -1,19 +1,45 @@
 <style>
-    svg:first-child {
-        display: none;
+    .u-hidden{
```

Comment:
I am a big supporter of util classes in HTML/CSS, although…

The purpose of this style was to hide the icon system itself, as it may occupy a few pixels at the top of the screen, depending on the browser.

If you change the selector, please also use it for the output of `{{{system}}}`, e.g. by wrapping it in an element with said class.

Reply:
Ok, `svg.svg-item` class requested below  override this pseudo selector so my change has no sense anymore.